aiida/backends/tests/orm/test_comments.py

```diff
@@ -73,6 +71,49 @@ def test_comment_collection_delete(self):
         with self.assertRaises(exceptions.NotExistent):
             Comment.objects.get(id=comment_pk)
 
+    def test_comment_collection_delete_all(self):
+        """Test deleting all Comments through the collection."""
+        comment = Comment(self.node, self.user, 'I will perish').store()
+        Comment(self.node, self.user, 'Surely not?').store()
+        comment_pk = comment.pk
+
+        # Assert the comments exist
+        self.assertEqual(len(Comment.objects.all()), 3)
+
+        # Delete all Comments
+        Comment.objects.delete_all()
+
+        with self.assertRaises(exceptions.NotExistent):
+            Comment.objects.delete(comment_pk)
+
+        with self.assertRaises(exceptions.NotExistent):
+            Comment.objects.get(id=comment_pk)
+
+    def test_comment_collection_delete_many(self):
```

Comment:
Is there a test to check that passing empty filters raises?

Reply:
Not here, but under `.implementation.test_comments` there is. Do you want one here as well?